demo/installer/x86_64/install.sh

```diff
@@ -13,7 +13,7 @@ cd $(dirname $0)
 echo "Demo Installer: platform: $platform"
 
 # Install demo on same block device as ONIE
-blk_dev=$(blkid | grep ONIE-BOOT | awk '{print $1}' | sed -e 's/[0-9]://' | head -n 1)
+blk_dev=$(blkid | grep ONIE-BOOT | awk '{print $1}' |  sed -e 's/[0-9]:$//' | sed -e 's/\([0-9]\)\(p\)/\1/' | head -n 1)
```

Comment:
yak shave -- I don't think you need the "$" in the first sed as the ":" already anchors the search.  no big deal though.

also you do not need to pipe sed into another sed.  instead you can specify multiple "-e" expressions, which are applied one after another in order.  again, no big deal.

do not change anything.